### DISCUSSION TITLE
Why not add an auto-login middleware feature?

### DISCUSSION THREAD

**Discussion by Peterragheb** [spatie/laravel-login-link#8] [Ideas]
The main purpose of this package, as I understand it, is to save time for developers.

Then I'm suggesting adding an auto-login middleware feature that enables the user to be logged in automatically without having to click on the login button in the login form.

What do you think?

**Comment by freekmurze** [comment]
I think it's nice to still be able to see the login screen itself, so I'm going to pass on adding a middleware for now.
